.github/styles/Microsoft/Avoid.yml

```diff
@@ -6,7 +6,6 @@ level: error
 tokens:
   - abortion
   - and so on
-  - and/or
```

Comment:
no, sorry. this is not grammatically correct and we can't accept it.

Reply:
Done.